go/tools/fetch_repo/fetch_repo_test.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+
+	"golang.org/x/tools/go/vcs"
+)
+
+var (
+	root = &vcs.RepoRoot{
+		VCS:  vcs.ByCmd("git"),
+		Repo: "https://github.com/bazeltest/rules_go",
+		Root: "github.com/bazeltest/rules_go",
+	}
+)
+
+func TestGetRepoRoot(t *testing.T) {
+	for _, tc := range []struct {
```

Comment:
Please add a stub for `vcs.RepoRootForImportPath` since it accesses the network. Gazelle does this in [resolve_external_test.go](https://github.com/bazelbuild/rules_go/blob/master/go/tools/gazelle/rules/resolve_external_test.go).

This might make the tests a little tautological, but I guess the important thing is to verify that the function is called when `cmd` and `remote` aren't specified.